plugins/org.python.pydev.core/src/org/python/pydev/core/IInterpreterInfo.java

```diff
@@ -96,6 +96,8 @@
 
     public Properties getStringSubstitutionVariables();
 
+    public Properties getStringSubstitutionVariables(boolean addEnvironmentVariables);
```

Comment:
I think you shouldn't add a new method, just change the existing one (and then make callers decide on whether they want `true` or `false` there).